src/ir/annotations.rs

```diff
@@ -134,20 +134,23 @@ impl Annotations {
         if comment.kind() == CXComment_HTMLStartTag &&
            comment.get_tag_name() == "div" &&
            comment.get_num_tag_attrs() > 1 &&
-           comment.get_tag_attr_name(0) == "rustbindgen" {
+           comment.get_tag_attr_name(0) == Some("rustbindgen".to_string()) {
```

Comment:
Good catch, thank you! I had to use as_ref().map(|s| s.as_str()) to prevent the compiler from complaining about seeing `&String` and expecting `&'static str`.